NOTICE

```diff
@@ -4,3 +4,11 @@ Copyright 2017 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+This product includes software from the Arrow project (Apache 2.0)
+https://github.com/apache/arrow
+
+
+This product includes software from the LLVM project
+ * distributed under the University of Illinois Open Source
```

Comment:
This should anyway go inside the LICENSE file, but I don't see where this code actually was imported

Reply:
@merlimat We use `clang-format` for code-formatting.

Reply:
Seems that we don't need to notify this.
http://llvm.org/docs/DeveloperPolicy.html#id27